filesystem/uimage.ksy

```diff
@@ -0,0 +1,103 @@
+meta:
+  id: u_image
+  endian: be
+seq:
+  - id: header
+    type: u_header
+  - id: data
+    size: header.image_size
+types:
+  u_header:
+    seq:
+      - id: magic
+        contents: [0x27, 0x05, 0x19, 0x56]
+      - id: header_crc
+        type: u4
+      - id: timestamp
+        type: u4
+      - id: image_size
+        type: u4
+      - id: load_address
+        type: u4
+      - id: entry_address
+        type: u4
+      - id: data_crc
+        type: u4
+      - id: os_type
+        type: u1
+        enum: os
+      - id: architecture
+        type: u1
+        enum: arch
+      - id: image_type
+        type: u1
+        enum: t
+      - id: compression_type
+        type: u1
+        enum: comp
+      - id: name
+        size: 32
+        encoding: UTF-8
+        type: strz
+enums:
+  os:
+    0:  invalid   # Invalid OS 
+    1:  openbsd   # OpenBSD  
+    2:  netbsd    # NetBSD 
+    3:  freebsd   # FreeBSD  
+    4:  bsd4_4    # 4.4BSD 
+    5:  linux     # Linux  
+    6:  svr4      # SVR4   
+    7:  esix      # Esix   
+    8:  solaris   # Solaris  
+    9:  irix      # Irix   
+    10: sco       # SCO    
+    11: dell      # Dell   
+    12: ncr       # NCR    
+    13: lynxos    # LynxOS 
+    14: vxworks   # VxWorks  
+    15: psos      # pSOS   
+    16: qnx       # QNX    
+    17: u_boot    # Firmware 
+    18: rtems     # RTEMS  
+    19: artos     # ARTOS  
+    20: unity     # Unity OS 
+    21: integrity # INTEGRITY  
```

Comment:
Please consider using [verbose enums](http://doc.kaitai.io/user_guide.html#verbose-enums) to move these extra documentation strings from YAML comments to something that will actually lead to generation of documentation in target languages.